CONTRIBUTING.md

```diff
@@ -5,20 +5,91 @@
 I'm glad you're reading this, I really appreciate the effort you're
```

Comment:
Yeah btw since it's a company-wide library, it's better to use "we".

Reply:
Good point.